src/main/java/net/minecraftforge/registries/DeferredRegister.java

```diff
@@ -30,119 +30,85 @@
 import java.util.function.Supplier;
 
 /**
- * Utility class to help with managing registry entries.
- * Maintains a list of all suppliers for entries and registers them during the proper Register event.
- * Suppliers should return NEW instances every time.
+ * Utility class to help with managing registry entries.<br>
+ * Maintains a list of all suppliers for entries and registers them during the proper event.<br>
+ * Suppliers should return new instances every time.
+ * <p>
+ * Example Usage:
+ * <code><pre>
+ * private static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, MODID);
+ * private static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, MODID);
  *
- *Example Usage:
- *<pre>{@code
- *   private static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, MODID);
- *   private static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, MODID);
+ * public static final DeferredHolder<Block> ROCK_BLOCK = BLOCKS.register("rock", () -> new Block(Block.Properties.create(Material.ROCK)));
+ * public static final DeferredHolder<Item> ROCK_ITEM = ITEMS.register("rock", () -> new BlockItem(ROCK_BLOCK.get(), new Item.Properties().group(ItemGroup.MISC)));
  *
- *   public static final RegistryObject<Block> ROCK_BLOCK = BLOCKS.register("rock", () -> new Block(Block.Properties.create(Material.ROCK)));
- *   public static final RegistryObject<Item> ROCK_ITEM = ITEMS.register("rock", () -> new BlockItem(ROCK_BLOCK.get(), new Item.Properties().group(ItemGroup.MISC)));
- *
- *   public ExampleMod() {
- *       ITEMS.register(FMLJavaModLoadingContext.get().getModEventBus());
- *       BLOCKS.register(FMLJavaModLoadingContext.get().getModEventBus());
- *   }
- *}</pre>
+ * public ExampleMod() {
+ *     ITEMS.register(FMLJavaModLoadingContext.get().getModEventBus());
+ *     BLOCKS.register(FMLJavaModLoadingContext.get().getModEventBus());
+ * }
+ * </pre></code>
  *
  * @param <T> The base registry type
  */
+@SuppressWarnings("deprecation") // TODO 1.20.2+ - Find+Replace RegistryObject with DeferredHolder.
 public class DeferredRegister<T>
 {
     /**
-     * DeferredRegister factory for forge registries that exist <i>before</i> this DeferredRegister is created.
+     * Factory method for DeferredRegister. The target registry does not need to exist yet.<br>
```

Comment:
My opinion is that <br> shouldn't be mixed with <p> tags in the same javadoc. Additionally, this is causing 2 blank lines for no reason, as it stands. I suggest deleting this one <br> tag.